shell/browser/web_view_guest_delegate.cc

```diff
@@ -39,6 +39,11 @@ void WebViewGuestDelegate::AttachToIframe(
             content::WebContents::FromRenderFrameHost(embedder_frame));
 
   content::WebContents* guest_web_contents = api_web_contents_->web_contents();
+
+  // Force a refresh of the webPreferences so that OverrideWebkitPrefs runs on
+  // the new web contents before the renderer process initializes.
+  guest_web_contents->NotifyPreferencesChanged();
```

Comment:
Not super confident in this change, but I think OverrideWebkitPrefs is happening earlier now, due to the referenced CL, before we have the WebPreferences associated with the WebContents. Needs some more study to make sure this isn't undue work.